### ISSUE TITLE
remove classes Minilogger and Pipe

### ISSUE THREAD

**Issue by gogo2464** [masthoon/pwintools#6]
Pipe() could be replaced by os.pipe and Minilogger could be repalced by print or another built in function. This code is useless.

**Comment by masthoon** [comment]
I agree that we could replace Pipe object with os.pipe (+set_inheritable for subprocess) if it works.
Regarding Minilogger, I disagree, logging has many more advantages than using print (filtering, line + context info, easy to redirect to socket/file), I don't think we should remove it.

**Comment by gogo2464** [comment]
Then we could use 

```
import logging
logging.info("info")
logging.debug("debug")
logging.error("error")
```

it is a built in function.
